src/Chip.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.16;
+
+import "solmate/tokens/ERC20.sol";
+
+contract Chip is ERC20 {
```

Comment:
are there any "this is just for fun" sort of disclaimers we should put on here?

Reply:
I like that idea

Reply:
Drafted this on the NatSpec. Can refine